atomsci/ddm/pipeline/diversity_plots.py

```diff
@@ -484,7 +483,7 @@ def _compare_solubility_datasets(ecfp_radius=6):
     for i, mol in enumerate(base_mols):
         if mol is None:
             print('Unable to get base molecule for compound %d = %s' % (i, del_compound_ids[i]))
-    base_smiles = [Chem.MolToSmiles(mol) for mol in base_mols]
+    _base_smiles = [Chem.MolToSmiles(mol) for mol in base_mols]
```

Comment:
delete this function and other dataset-specific ones

Reply:
Deleted https://github.com/ATOMScience-org/AMPL/commit/6227792ae02fc2b634b6fe45992f2f8b27d44999